bundle/src/main/java/com/adobe/acs/commons/ondeploy/impl/OnDeployExecutorImpl.java

```diff
@@ -226,11 +229,14 @@ protected void runScripts(ResourceResolver resourceResolver, List<OnDeployScript
         }
     }
 
-    protected void trackScriptEnd(Resource statusResource, String status) {
+    protected void trackScriptEnd(Resource statusResource, String status, String output) {
         try {
             ModifiableValueMap properties = statusResource.adaptTo(ModifiableValueMap.class);
             properties.put(SCRIPT_STATUS, status);
             properties.put(SCRIPT_DATE_END, Calendar.getInstance());
+            if (output != null) {
```

Comment:
This should just be `properties.put(SCRIPT_OUTPUT, output)` even if the `output` is null because you would want a successful re-run of a script to overwrite the field if it currently has a value in it.

Reply:
Applied fix